### ISSUE TITLE
[ddddr] remove minuit dependancy (Trac #2008)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2000]
either replace it with the implementation of minuit in lilliput or use the
gulliver framework

Migrated from https://code.icecube.wisc.edu/ticket/2008
```json
{
    "status": "closed", 
    "changetime": "2019-03-18T21:35:31", 
    "description": "either replace it with the implementation of minuit in lilliput or use the\ngulliver framework", 
    "reporter": "kjmeagher", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1552944931277334", 
    "component": "combo reconstruction", 
    "summary": "[ddddr] remove minuit dependancy", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2017-05-09T17:10:21", 
    "milestone": "Vernal Equinox 2019", 
    "owner": "mjl5147", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20170920T15:24:10: jgonzalez commented:

> And by the way... this is causing trouble in Ubuntu 16.04. Related to this:
> https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=552226

**Comment by icecube-trac** [comment]
Trac update at 20180613T18:26:46: kjmeagher changed milestone from "IceRec 5.2" to "Autumnal Equinox 2018"

**Comment by icecube-trac** [comment]
Trac update at 20181001T21:58:22: olivas changed milestone from "Autumnal Equinox 2018" to "Winter Solstice 2018"

**Comment by icecube-trac** [comment]
Trac update at 20190311T16:16:56: 
* olivas changed owner from "jgonzalez" to ""
* olivas changed status from "new" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20190311T20:12:05: 
* mjl5147 changed status from "assigned" to "accepted"
* mjl5147 changed owner from "" to "mjl5147"

**Comment by icecube-trac** [comment]
Trac update at 20190318T21:35:31: 
* mjl5147 changed status from "accepted" to "closed"
* mjl5147 commented:

> In changeset 172125:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="172125"
> Switched from Minuit2 to simplex for the minimization. Also removed the extraneous classes that aren't needed any longer. This removes the ROOT/minuit dependence and closes #2000.
> ```


* mjl5147 changed resolution from "" to "fixed"
